### ISSUE TITLE
Rule Change: quotes for import with

### ISSUE THREAD

**Issue by GrosSacASac** [eslint/eslint#20820]
### What rule do you want to change?

quotes

### What change do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```js
"quotes": [error, "backtick", { "avoidEscape": false }],



import packageJson from "./package.json" with { type: "json" };
```

### What does the rule currently do for this code?

  14:55  error  Strings must use backtick  quotes

### What will the rule do after it's changed?

Do not recommend backticks inside import

### Participation

- [ ] I am willing to submit a pull request to implement this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

Backticks are not valid inside imports I believe
Also the lint --fix, breaks the parsing of the next lint --fix

Not a big deal, but I think it should be changed

**Comment by Tanujkanti4441** [comment]
Hi @GrosSacASac, thanks for the issue, but the [quotes](https://eslint.org/docs/latest/rules/quotes) rule is deprecated, so we can't make changes to it.
